apps-client/mobile/app/(tabs)/(official)/index.tsx

```diff
@@ -30,13 +31,29 @@ import PPLEIcon from '@app/assets/pple-icon.svg'
 import { UserAddressInfoSection } from '@app/components/address-info'
 import { AnnouncementCard, AnnouncementCardSkeleton } from '@app/components/announcement'
 import { ElectionCard } from '@app/components/election/election-card'
+import { RefreshControl } from '@app/components/refresh-control'
 import { SafeAreaLayout } from '@app/components/safe-area-layout'
 import { reactQueryClient } from '@app/libs/api-client'
+import { useSession } from '@app/libs/auth'
 
 export default function OfficialPage() {
+  const queryClient = useQueryClient()
+  const onRefresh = useCallback(async () => {
+    await Promise.all([
+      queryClient.invalidateQueries({
```

Comment:
```suggestion
      queryClient.resetQueries({
```

Reply:
left this as invalidate bc if reset the data is lost
and the section doesn't have loading state so it will blink and reappear